search-resources-api/src/test/java/no/unit/nva/search/ConstantsTest.java

```diff
@@ -0,0 +1,15 @@
+package no.unit.nva.search;
+
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+
+public class ConstantsTest {
```

Comment:
I don't think we need this the constants are likely tested elsewhere if they are in use